addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3125,8 +3118,31 @@ options.registry.WebsiteAnimate = options.Class.extend({
         if (widgetName === 'no_animation_opt') {
             return !this.isAnimatedText;
         }
+        if (widgetName === 'animation_launch_opt') {
+            return !this.$target[0].closest('.dropdown');
+        }
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    _computeVisibility(methodName, params) {
+        if (this.$target[0].matches('img')) {
+            return isImageSupportedForStyle(this.$target[0]);
+        }
+        return this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _onImageChanged(ev) {
+        return this.updateUI();
+    },
```

Comment:
We cannot do that, the async part of updateUI should be considered. And I wonder why this should be something specific to the Animate option: why don't we need this for other options? If the image is changed, aren't all options destroyed and recreated anyway? :thinking: